tests/integration/targets/activation/tasks/main.yml

```diff
@@ -172,14 +201,296 @@
         that:
           - _result.changed
 
+    - name: Copy rulebook activation
+      ansible.eda.rulebook_activation:
+        name: "{{ activation_name }}-copy"
+        copy_from: "{{ activation_name }}"
+        project_name: "{{ project_name }}"
+        rulebook_name: "{{ rulebook_name }}"
+        decision_environment_name: "{{ decision_env_name }}"
+        organization_name: Default
+      register: _result
+
+    - name: Check rulebook activation update
+      assert:
+        that:
+          - _result.changed
+
+    - name: Get information about rulebook activation
+      ansible.eda.rulebook_activation_info:
+      register: _result_activation_info
+
+    - name: Assert that copied rulebook activations are equal except name
+      ansible.builtin.assert:
+        that:
+          - _result_activation_info.activations[0].description  ==
```

Comment:
The test assumes that there are not other activation created. This is a bad practice. It should fetch specifically the expected activations and not depend on the position in the list.